documentation/restore.md

```diff
@@ -0,0 +1,26 @@
+# Restore Puppet Enterprise using the PEADM module
+
+Once you have a [backup](backup.md) you can restore a PE primary server. It is important to highlight that you can not use the `peadm::restore` plan with a backup that was not created with the `peadm::backup` plan.
```

Comment:
Can `peadm::backup` be only used on the primary, not on other servers? Does it also work in the XL architecture?
Does the user need to know something about what will happen after they call the plan? For example, will there be any downtime?
Will it timestamp backups, overwrite previous ones, etc?

Reply:
It should only be run from Primary and it will work on XL by contacting the PuppetDB database in PostgreSQL and doing a remote backup to Primary, this requires temporary escalation in PostgreSQL. 

It will backup to a timestamp so should not overwrite

It stops services so there will be downtime and if restoring databases it will reinitialize replicas, which as viewed as the safest way to ensure databases were correct. 

We probably should also note as it uses peadm::get_targets function it needs the services to be up and running at the start of either restore or backup

Reply:
I implemented some changes based on this suggestion, starting on line 5.

https://github.com/puppetlabs/puppetlabs-peadm/pull/294/files#diff-e9069702983897147b48e019b1eb79a368506db0d32e09a65c7cfac83f511df6R5